power/nut-client/nut-client.yaml

```diff
@@ -5,6 +5,7 @@ depends:
     - addresses
     - connectivity
     - etcfiles
+  - path: /system/run/machined/machine.sock
```

Comment:
also add a wait so extension starts after file is present, otherwise in rare situation extension service creates a folder with the mount source path